OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -280,7 +285,7 @@ public void setIsVideosCacheRestored(boolean restored) {
     }
 
     /**
-     * Clears all the shared preferences being used in app.
+     * Clears all the shared preferences that are being used in app.
```

Comment:
Another small grammer fix: it should be "in **the** app". Also, "being" could be removed without losing any essential meaning here, I suppose.